kernel/src/sched/priority_scheduler.rs

```diff
@@ -13,12 +14,20 @@ use ostd::{
     trap::disable_local,
 };
 
-use super::priority::{Priority, PriorityRange};
+use super::{
+    priority::{Priority, PriorityRange},
+    stats::{inject_scheduler_stats, SchedulerStats},
+};
 use crate::{prelude::*, thread::Thread};
 
 pub fn init() {
     let preempt_scheduler = Box::new(PreemptScheduler::default());
     let scheduler = Box::<PreemptScheduler<Thread, Task>>::leak(preempt_scheduler);
+
+    // Inject the scheduler into the system for statistics.
+    inject_scheduler_stats(scheduler);
```

Comment:
We have used the word "inject" to refer to something not implemented by OSTD but set as a system-wide singleton to make part of the OSTD's job work. In this case I think it is better phrased by "`set_stats_from_scheduler`".